openedx/adg/lms/student/helpers.py

```diff
@@ -36,7 +37,7 @@ def compose_and_send_adg_activation_email(user, activation_key):
         'activation_link': activation_url
     }
 
-    MandrillClient().send_mandrill_email(MandrillClient.USER_ACCOUNT_ACTIVATION, user.email, context)
+    task_send_mandrill_email.delay(MandrillClient.USER_ACCOUNT_ACTIVATION, user.email, context)
```

Comment:
Please verify it, I could be wrong but you need to pass serializable object to celery task everywhere